### ISSUE TITLE
Question on training setup (8 GPU), supported base models, and reward curves for replication

### ISSUE THREAD

**Issue by h6kplus** [franklinz233/Astrolabe#5]
Hi, thanks for realeasing this awesome work! 

I’m very interested in reproducing the results from Astrolabe and exploring extensions with customized reward functions. My GPU is limited so I'm only able to conduct training on 8 GPUs. From the repo I see that the 8 gpu setting is 3x smaller in batch size compared to 48GPU. I'm wondering wil this setting cause the model to fail to converge and see reward going up? Cause I trained for ~20 steps on longlive with HPSV3 and saw the rewrad fluctuating around 0.4 so I'm a little concerned. I've attached the training curve for your reference.

Also under the 8GPU setting, is there any starting model (self-forcing, longlive, causal-forcing) that may be more robust in training? Thank you!

Thanks again for releasing this work and it is really inspiring! Looking forward to reproduce that and try to use.

<img width="519" height="283" alt="Image" src="https://github.com/user-attachments/assets/a9ac508b-63be-4d5c-959c-10b29613116f" />

**Comment by franklinz233** [comment]
Hi @h6kplus ~
Thank you for your kind words and for your interest in reproducing Astrolabe! It's great to hear you're exploring extensions with customized reward functions.

Regarding your concerns about training with 8 GPUs: yes, we have successfully tested our method on both 8 and 16 GPU configurations. While the smaller batch size means each epoch takes longer to complete, the training does converge stably. 
The key is to monitor the correct metric: in wandb, you should track `eval_reward_video_<reward_name>` (e.g., `eval_reward_video_hpsv3`), not the `reward_std_mean` metric. 

For stability on limited hardware, we recommend:
1. Starting from Causal Forcing – in our experience, this checkpoint leads to more stable training dynamics.
2. If using LongLive, initialize with its LoRA adapter rather than loading the base weights alone; this tends to be more stable.
3. Keep configuration values like `num_image_per_prompt` and `num_groups` at the larger settings in our provided configs for better training stability.

I can share a reference reward curve from a Causal Forcing training run on 8 GPUs if that would be helpful. 

<img width="359" height="289" alt="Image" src="https://github.com/user-attachments/assets/20da199f-5ecd-43e4-9dae-1f3b9238659c" />

<img width="358" height="277" alt="Image" src="https://github.com/user-attachments/assets/d7d3be28-62a7-4e05-9d47-117427a2e6aa" />

**Comment by h6kplus** [comment]
Thanks for your help! Our work is now out at  [`https://phy-motion.github.io/`](https://phy-motion.github.io/)
